.travis.yml

```diff
@@ -23,6 +24,14 @@ matrix:
       gemfile: gemfiles/rails_5.1.gemfile
     - rvm: truffleruby
       gemfile: gemfiles/rails_5.2.gemfile
+    - rvm: truffleruby
+      gemfile: gemfiles/rails_6.0.gemfile
+    # rails-4.2 does not support ruby-2.6
```

Comment:
Don't think this is needed. Let me try to restore the previous state and see if the build passes?